meshtastic/tests/test_examples.py

```
@@ -52,3 +63,97 @@ def test_examples_hello_world_serial_with_arg(
+    assert (
+        EXAMPLE_CONFIG_PATH.exists()
+    ), f"Example config not found: {EXAMPLE_CONFIG_PATH}"
+    config = _require_mapping(
+        yaml.safe_load(EXAMPLE_CONFIG_PATH.read_text(encoding="utf-8")),
+        label="example_config.yaml",
+    )
+
+    owner = config.get("owner")
+    owner_short = config.get("owner_short")
+    channel_url = config.get("channel_url")
+    assert isinstance(owner, str) and owner.strip()
```

Ruff (0.15.4)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)